components/Forms/ComponentFactory.tsx

```diff
@@ -193,10 +193,7 @@ export const ComponentFactory: React.VFC = observer(({}) => {
         <NeedHelp title={tsln.needHelp} links={root.summary.needHelpLinks} />
       </div>
 
-      <div className="mt-16">
-        <h2 className="h2 text-content mb-8">{tsln.faq}</h2>
-        <FAQ />
-      </div>
+      <FAQ />
```

Comment:
for accessibility this should be an abbr

Reply:
(will follow up on this, just want to merge the PR now as it affects a bunch of files)

Reply:
what do you mean should be an abbr? I'm not familiar with that term in this context.

Reply:
nevermind I thought it was text but it is a component